libnmstate/_error.py

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+
+class NmstateError(Exception):
+    BUG = -1
+    NOT_SUPPORT = 95
```

Comment:
IMHO this needs to be either `NOT_SUPPORTED` or `NO_SUPPORT`. Why the number 95?

Reply:
95 is the common C error code for no support:

```
[fge@Gris-Laptop ~]$ errno 95
EOPNOTSUPP 95 Operation not supported
```

Reply:
Changed to `NO_SUPPORT`.